### DISCUSSION TITLE
Compatible pumps in Australia

### DISCUSSION THREAD

**Discussion by mcvax** [tagyoureit/nodejs-poolController#799] [Q&A]
Hi everyone

I'm in the process of building a pool and am looking to find a pump that would be compatible with a Nixie system in Australia.

I've looked at the list of recommended pumps here, but although Pentair list the Intelliflo models on their website I can't find anyone that stocks them in Australia.
Recommended: https://github.com/tagyoureit/nodejs-poolController/discussions/523
Pentair:  https://www.pentairpool.com.au/185/Pumps/Filtration-Pumps

Has anyone found a pump in Australia that integrates with Nixie?

Thanks

Chris

**Comment by rstrouse** [comment]
The Hayward VS also work.  Pentair also markets under Sta-Rite

**Reply by mcvax** [reply]
Great thank you.  The Hayward TriStar VS TS540 is readily available so hopefully that will work.

**Reply by rstrouse** [reply]
Is that the same as the vs900 or vs700?

**Reply by mcvax** [reply]
I really don't know.  They seem to assemble or build pumps in this region and market them under the same line/brand and the North American version.  But they appear to be very different.

The VS700 looks like a different beast to the TS540VS.  On the plus side the TS540VS appears to have a modbus interface, something that almost no other pump in the Australian market has. 

**VS700** 
https://www.hayward-pool.com/shop/en/pools/super-pump-vs-i-pmspvs--1

**TriStar TS540VS**
https://www.hayward-pool.com.au/shop/en/australia/aus-pool-pumps/tristar-vs
